docs/types/conversion.rst

```diff
@@ -99,6 +99,26 @@ rules explicit::
     uint8 d = uint8(uint16(a)); // d will be 0x34
     uint8 e = uint8(bytes1(a)); // e will be 0x12
 
+``bytes`` arrays and ``bytes`` calldata slices can be converted to fixed bytes types(``bytes1``/.../``bytes32``).
+If array is longer than the target fixed bytes type panic will happen.
+If array is shorter than the target type, value will be padded on the right side.
+
+::
+
+    // SPDX-License-Identifier: GPL-3.0
+    pragma solidity >0.8.2;
```

Comment:
```suggestion
    pragma solidity ^0.8.4;
```

Reply:
I need to use ``pragma solidity >0.8.3;` because of cmdLineTests script fails with `^0.8.4`

Reply:
I thought we fixed that?

Reply:
I don't remember if someone worked on that, but let me check if I can fix it quickly, so we can set proper pragmas here

Reply:
I fixed that in #10770.

Note that it makes the check ignore the version returned from `get_version.sh`. If you haven't rebased on `develop` after yesterday's release then it will still see `0.8.3` (rather than `0.8.4`) as the current dev version.

Reply:
I implemented what was needed for `cmdlineTests.sh` in order to be able to pass when we change to `^0.8.4`
https://github.com/ethereum/solidity/pull/11166